### ISSUE TITLE
Show HN and Ask HN stories don't mark as read when clicking comments bubble

### ISSUE THREAD

**Issue by tedmiston** [jingweno/hacker-menu#32]
I think that they should behave this way because, in this case of these post types, clicking the comments bubble opens the same link as clicking on the text itself.

**Comment by lokywin** [comment]
This is not always the case, is it? in the example screenshot below taken from **Show**
![screen shot 2015-10-16 at 1 26 38 pm](https://cloud.githubusercontent.com/assets/545713/10552585/af1f22be-7409-11e5-90aa-31a1d8ec4b67.png)
The heading link and comment link point to different places.

**Comment by tedmiston** [comment]
@lokywin Whoops, you're right -- only always on Ask HN posts.  I'll correct the title.

**Comment by jingweno** [comment]
The current behavior is [only clicking the link to the post will count as read (clicking other links like comments, rank etc. won't be counted as read)](https://github.com/jingweno/hacker-menu/blob/master/src/client/story.js#L12-L26). However, I see the point of clicking all links should be counted as read. Typically, users click the comment link to jump to the HN post and click the post link from the HN site, instead from the Hacker Menu app.

**Comment by tedmiston** [comment]
I agree completely.  I think this case is different because the post link and the HN post are the same for Ask HN.
